skimage/data/__init__.py

```diff
@@ -384,3 +385,34 @@ def stereo_motorcycle():
     return (load("motorcycle_left.png"),
             load("motorcycle_right.png"),
             np.load(_os.path.join(data_dir, "motorcycle_disp.npz"))["arr_0"])
+
+
+def hdr_images():
+    """
+    Bracket images useful for High Dynamic Range (HDR) imaging.
```

Comment:
A collection of images with varying exposure, useful for High Dynamic Range (HDR) imaging.